src/sqrt.ts

```diff
@@ -1,5 +1,8 @@
+import log2 from './log2';
+
 function sqrtPos(x: bigint): bigint {
-  var a = x, b = x+1n;
+  const initialGuess = 1n << (log2(x) / 2n + 1n);
```

Comment:
This can be `1n << ((1n + log2(x)) / 2n)`?

Reply:
@wolfram77 , not sure, this could make it less than mathematical value